app/search/search.component.ts

```diff
@@ -279,6 +282,17 @@ export class SearchComponent {
     return '';
   }
 
+  getStatus() {
```

Comment:
(see comment above)

is this necessary? or providing the i18n ids is already sufficient?